### PR TITLE
vimv: init at unstable-2019-10-31

### PR DESCRIPTION
**Pull request by kmein**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/vimv/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{ stdenv, bash, fetchFromGitHub }:
+stdenv.mkDerivation {
+  name = "vimv";
```
Comment (Lassulus) [inline pkgs/tools/misc/vimv/default.nix:3]:
```suggestion
  pname = "vimv";
```

File: pkgs/tools/misc/vimv/default.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{ stdenv, bash, fetchFromGitHub }:
+stdenv.mkDerivation {
+  pname = "vimv";
```
Comment (Lassulus) [inline pkgs/tools/misc/vimv/default.nix:3]: uhm, there should be a version :D

Comment (kmein) [reply pkgs/tools/misc/vimv/default.nix:3]: if there isn't one upstream, should i use the date of the last commit or the beginning of the commit hash?

Comment (Lassulus) [reply pkgs/tools/misc/vimv/default.nix:3]: example: `version = "unstable-2019-23-24"`
date is date of commit

File: pkgs/tools/misc/vimv/default.nix [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  src = fetchFromGitHub {
+    owner = "thameera";
+    repo = "vimv";
+    rev = "4152496c1946f68a13c648fb7e583ef23dac4eb8";
+    sha256 = "1fsrfx2gs6bqx7wk7pgcji2i2x4alqpsi66aif4kqvnpqfhcfzjd";
+  };
+  phases = [ "installPhase" ];
+  installPhase = ''
+    mkdir -p $out/bin
+    sed 's:#!/bin/bash:#!${bash}/bin/bash:' $src/vimv > $out/bin/vimv
```
Comment (Lassulus) [inline pkgs/tools/misc/vimv/default.nix:14]: does `patchShebangs $out/bin/vimv` work?

**Pull request merged**